Source/Classes/SLKTextInputbar.m

```diff
@@ -535,11 +538,8 @@ - (void)slk_didChangeTextViewContentSize:(NSNotification *)notification
 
 - (void)slk_setupViewConstraints
 {
-    UIImage *leftButtonImg = [self.leftButton imageForState:UIControlStateNormal];
-    
```

Comment:
Are they needed there? The leftVerMargin generated from the UIImage size is not used when the constraints are set. It looks like the real adjustment is made in slk_updateConstraintConstants. Not sure, this was something I noticed while implementing the proper fix, not really needed for it to work.